CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt

```diff
@@ -78,7 +79,17 @@ class GalleryFragment internal constructor() : Fragment() {
             inflater: LayoutInflater,
             container: ViewGroup?,
             savedInstanceState: Bundle?
-    ): View? = inflater.inflate(R.layout.fragment_gallery, container, false)
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_gallery, container, false)
+
+        //Checking media files list
+        if (mediaList.size == 0) {
+            view.findViewById<ImageButton>(R.id.delete_button).isEnabled = false
+            view.findViewById<ImageButton>(R.id.share_button).isEnabled = false
+        }
+
+        return view
+    }
```

Comment:
I would suggest moving this to `onViewCreated` and to take advantage of Kotlin syntax e.g. `isEmpty()`